src/debugger-view.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import { useFormikContext } from 'formik';
 
+const replaceUndefinedWithNull = (k: string, v: any): unknown => {
```

Comment:
1. Consider adding this function to `src/utils` so that it can be imported and reused in other files if needed.
2. Consider using TSDoc comments to describe the function and its params
3. Consider renaming the params to be more descriptive to improve readability